### ISSUE TITLE
Likely installation error, but visualizer constantly throws exception

### ISSUE THREAD

**Issue by rhbourdeau** [zspitz/ExpressionTreeVisualizer#123]
**Describe the bug**  
Error thrown when attempting to use visualizer: "Unable to load the custom visualizer"

I assume this is a problem related to installation but I cannot figure out any other method of installation that works. 

**To Reproduce**  

1.  Perform installation as noted later in this issue.
2. Create a netcore console app with this sole statement in the Main body
_Expression<Func<string[], bool>> exp = o => o.Contains("e");_
3. Place a breakpoint on this statement.
4. Run
5. At breakpoint, step once past the statement, allowing it to execute. 
6.  Hover over exp variable.  
7. A floating window appears for this variable offering the "magnifying glass" to visualize the value of the variable. 
8. Select the menu pulldown and choose the "ExpressionTreeVisualizer" as an option. 
9. An error dialog is popped up.

**Expected behavior**  
I expected to see something like the screenshots shown in the Readme.md file when choosing the ExpressionTreeVisualizer to inspect an Expression while debugging.

**Screenshots**  
About to select the ExpressionTreeVisualizer
![selecting](https://user-images.githubusercontent.com/12534214/110989558-15186d00-8340-11eb-83d5-4804a6bc3216.jpg)

After selecting
![selectingerror](https://user-images.githubusercontent.com/12534214/110989565-18135d80-8340-11eb-8a5e-0d8ca7cb38b2.jpg)

**Version info:**

- Visual Studio version: 
Visual Studio 2019 Professional, Version 16.8.3

- Visualizer location: [VS install path, or Documents subfolder] 
For MANUAL installation I did this
Copied ZIP contents in "netcoreapp" folder, as is,
to C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\Packages\Debugger\Visualizers\netcoreapp
That should take care of debuggee.

Copied ZIP /ExpressionTreeVisualilzer.dll and /ExpressionTreeVisualizer/ folder 
to  C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\Packages\Debugger\Visualizers
That should take care of debugger.

I also attempted to perrform the installation via NuGet package. While nuget asserts that the package is installed for my project, the visualizer is not an option to inspect Expressions.

- Version: [GitHub release version or project commit]
v 1.7.110

**Additional context**  
Windows 10
C# Net Core 3.1 console application

**Comment by zspitz** [comment]
Firstly, thanks for a well-written and comprehensive bug report. It's not something that can be taken for granted.

Also, the NuGet package isn't intended for use as  debugging visualizer in VS; AFAIK debugging visualizers can only be installed by copying them into the appropriate folders.

What precise version of .NET Core are you using (`dotnet --version`; `dotnet --list-sdks`)? I am trying to reproduce on my machine.

Also, you didn't mention this, but you do need to unblock the ZIP file; instructions for doing so can be found [here](https://github.com/zspitz/ExpressionTreeVisualizer/wiki/Troubleshooting-visualizer-installation).

**Comment by rhbourdeau** [comment]
Aha! The installation instructions you provided in the README file mentioned to "Unblock the ZIP" but I had never heard that term before and just assumed you mean "unpack the files". I did not think to look at the Wiki. 

I "uninstalled" the visualizer, unblocked the entire ZIP file as you indicate in the Wiki, and reinstalled. That appears to have eliminated the problem and I am now able to invoke the visualizer while debugging.

Perhaps a direct link from the README installation instructions to the Wiki page regarding "unblocking" would be helpful for some folks.

Thanks, I'm off to work with your tool now. Much appreciated.

Side note, I'm actually working with Core in NET5. Just for reference here's my NET info, which I just realized I need to update.

> dotnet --version; dotnet --list-sdks
5.0.101
2.1.202 [C:\Program Files\dotnet\sdk]
2.1.509 [C:\Program Files\dotnet\sdk]
3.1.100 [C:\Program Files\dotnet\sdk]
5.0.101 [C:\Program Files\dotnet\sdk]
